generator/cmd/protoc-gen-gclient/main.go

```diff
@@ -0,0 +1,127 @@
+package main
```

Comment:
nit: this is missing the copyright boiler plate.

Reply:
done